jvcl/run/JvRollOut.pas

```diff
@@ -1451,13 +1454,19 @@ procedure TJvCustomRollOut.CheckChildVisibility;
       FChildControlVisibility.Sorted := True;
     end;
 
-    for I := 0 to ControlCount - 1 do
-      if (Controls[I] is TWinControl) and (TWinControl(Controls[I]).Visible) then
-      begin
-        FChildControlVisibility.AddObject(Controls[I].Name, Controls[I]);
-        if CollapseCtrlsOnButton or (TWinControl(Controls[I]).Top > ButtonHeight) then
-          TWinControl(Controls[I]).Visible := False;
-      end;
+    DisableAlign;
+
```

Comment:
To make it clearer that the `DisableAlign` is protected by the `try..finally` block, this empty line should be removed